.github/workflows/benchmark.yml

```diff
@@ -10,20 +10,109 @@ jobs:
       # Prevents other configs from being canceled if one of them fails.
       fail-fast: false
       matrix:
-        config_dataset_info:
-          [
+        config_dataset_info: [
             # config                   dataset    lookahead    img-extension  source  loader      max-res  share-intrinsics
-            [sift_front_end,           door-12,               12,  JPG,  test_data,  olsson-loader,  1296, true],
-            [deep_front_end,           door-12,               12,  JPG,  test_data,  olsson-loader,  1296, true],
-            [sift_front_end,           skydio-8,              8,   jpg,  gdrive ,    colmap-loader,  760, true],
-            [deep_front_end,           skydio-8,              8,   jpg,  gdrive,     colmap-loader,  760, true],
-            [sift_front_end,           skydio-32,             32,  jpg,  gdrive,     colmap-loader,  760, true],
-            [deep_front_end,           skydio-32,             32,  jpg,  gdrive,     colmap-loader,  760, true],
+            [
+              sift_front_end,
```

Comment:
this formatting makes it harder to read, can we revert to the way it was?

Reply:
Fixed